storage/kvstore/direct_access_devicekey/tests/TESTS/direct_access_devicekey/direct_access_devicekey_tdb/main.cpp

```diff
@@ -23,13 +23,13 @@
 #include <stdio.h>
 #include <string.h>
 #include "DeviceKey.h"
-#include "KVStore.h"
```

Comment:
File permission should not have changed.

Reply:
@evedon Revert to `644` pervious file permission and also rebased this PR to master. Please re-review